src/status_im2/contexts/communities/menus/community_options/view.cljs

```diff
@@ -41,12 +41,13 @@
    :on-press            #(hide-sheet-and-dispatch [:chat.ui/mark-all-read-in-community-pressed id])})
 
 (defn mute-community
-  [_ muted?]
+  [id muted?]
   {:icon                (if muted? :i/muted :i/activity-center)
    :accessibility-label (if muted? :unmute-community :mute-community)
    :label               (i18n/label (if muted? :t/unmute-community :t/mute-community))
    :sub-label           (when muted? (str "muted for 15 minutes"))
```

Comment:
you should probably adjust this string. how long is the community muted for btw?

fro. the designs i believe it's not just muted or unmuted, it's muted for 15 mins, 1 hour etc.. or is this something we will handle a different time?

Reply:
at the moment the muting of a community is not time locked I guess its right to remove the string and  handle that in a different PR

Reply:
ok cool, do you know if it's  possible to time lock on the back end? or it's a feature that will be added later?

Reply:
I will have to confirm that